draft-ietf-rtcweb-jsep.xml

```diff
@@ -1097,6 +1097,54 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           simulcast as described in
           <xref target="sec.simulcast" />).</t>
         </section>
+
+        <section title="stopTransceiver" anchor="sec.transceiver-stop">
+          <t>
+            The stopTransceiver method stops an RtpTransceiver.
+            See <xref target="sec.transceiver-stopped"/> for more details.
+          </t>
+        </section>
+        <section title="isTransceiverStopped" anchor="sec.transceiver-stopped">
+          <t>
+            The isTransceiverStopped method returns true if
+            the transceiver has been stopped, either by a call
+            to stopTransceiver or by a call to
+            setRemoteDescription, and false otherwise.
+          </t>
+          <t>
+            A stopped RtpTransceiver does not send any
+            outgoing RTP or RTCP or process any incoming RTP
+            or RTCP.
```

Comment:
I added a note about making a zero port.  

But what's this about inactive m= sections not being stopped?  That's new to me.